### ISSUE TITLE
PyYaml does not get installed with Wandb

### ISSUE THREAD

**Issue by Allezxandre** [wandb/client#809]
* Weights and Biases version: `wandb, version 0.8.22`
* Python version: `Python 3.7.4`
* Operating System: `Linux`

### Description

Installing WandB on a new Conda environment, and then importing `wandb` in Python fails with an import error for `Yaml`. 

Installing `PyYaml` afterwards fixes this, but it should have been installed with the first `pip` command.

### What I Did

```bash
> pip install wandb
[...]
Successfully installed Click-7.0 GitPython-3.0.5 chardet-3.0.4 configparser-4.0.2 docker-pycreds-0.4.0 gitdb2-2.0.6 gql-0.2.0 graphql-core-1.1 idna-2.8 nvidia-ml-py3-7.352.0 pathtools-0.1.2 promise-2.3 psutil-5.6.7 requests-2.22.0 sentry-sdk-0.14.1 shortuuid-0.5.0 smmap2-2.0.5 subprocess32-3.5.4 urllib3-1.25.8 wandb-0.8.22 watchdog-0.10.0
> python
Python 3.7.4 (default, Aug 13 2019, 20:35:49)
[GCC 7.3.0] :: Anaconda, Inc. on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import wandb
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/[...]/lib/python3.7/site-packages/wandb/__init__.py", line 45, in <module>
    from wandb.apis import InternalApi, PublicApi, CommError
  File "/home/[...]/lib/python3.7/site-packages/wandb/apis/__init__.py", line 97, in <module>
    from .file_stream import FileStreamApi
  File "/home/[...]/lib/python3.7/site-packages/wandb/apis/file_stream.py", line 11, in <module>
    from wandb import util
  File "/home/[...]/lib/python3.7/site-packages/wandb/util.py", line 24, in <module>
    import yaml
ModuleNotFoundError: No module named 'yaml'
> pip install PyYaml
[...]
Successfully installed PyYaml-5.3
> python
Python 3.7.4 (default, Aug 13 2019, 20:35:49)
[GCC 7.3.0] :: Anaconda, Inc. on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import wandb
>>>
```

**Comment by vanpelt** [comment]
I noticed this yesterday, @raubitsj it looks like whatever was pulling in
pyyaml in <3.7 isn’t happening in 3.7.  Let’s get a fix in the next release.

**Comment by raubitsj** [comment]
https://github.com/wandb/client/pull/814

Will be fixed in next release.  If you cant wait:
pip install --upgrade git+git://github.com/wandb/client.git@fix/add-back-yaml-depends#egg=wandb

**Comment by raubitsj** [comment]
This is fixed in 0.8.24
